cmd/operator/main.go

```diff
@@ -274,5 +264,11 @@ func getWatchNamespace() (string, error) {
 	if !found {
 		return "", fmt.Errorf("%s must be set", watchNamespaceEnvVar)
 	}
+	// WMCO can only function properly in this namespace, as OLM is unable to adjust the WICD RoleBinding to target a
+	// ServiceAccount in any namespace but this one.
+	supportedNamespace := "openshift-windows-machine-config-operator"
+	if ns != supportedNamespace {
+		return "", fmt.Errorf("the operator must be deployed in the %s namespace", supportedNamespace)
+	}
```

Comment:
You cannot just add this to `getWatchNamespace()`. You have now changed the meaning of the function. Like I mentioned before add an explicit `checkNamespace()` method to cleary call this out. We don't want this inlined (as simple as it might be) or buried in another method.

Reply:
I can even live with inlining if the comment is retained.